dashboards/resources/namespace.libsonnet

```diff
@@ -20,7 +20,7 @@ local template = grafana.template;
       template.new(
         name='namespace',
         datasource='$datasource',
-        query='label_values(kube_namespace_created{%(kubeStateMetricsSelector)s, %(clusterLabel)s="$cluster"}, namespace)' % $._config,
+        query='label_values(kube_namespace_status_phase{phase="Active", %(kubeStateMetricsSelector)s, %(clusterLabel)s="$cluster"}, namespace)' % $._config,
```

Comment:
What would happen if don't use `phase` here?

Reply:
Otherwise `Terminating` namespace also would be listed. I'm not sure `label_values` will dedupe.

Reply:
AFAIK it does dedup. However in this case ns cannot be in Active and Terminating state at the same time, so there is no duplicate. 
I am more curious about user experience here. For example, ns can be in a `Terminating` state, but workload can still be running (slowly shutting down). IMHO in such case it would be good to include `Terminating` ns in the selector dropdown

Reply:
Agree with you. Let me remove the `phase="Active` selector.